code/modules/mob/living/carbon/human/human.dm

```diff
@@ -766,8 +758,21 @@
 			if(S.siemens_coefficient <= 0)
 				total_coeff -= 0.95
 		siemens_coeff = total_coeff
-
-	return ..(shock_damage, source, siemens_coeff, def_zone, override, tesla_shock)
+	else if(!safety)
+		var/gloves_siemens_coeff = 1
+		var/species_siemens_coeff = 1
+		if(gloves)
+			var/obj/item/clothing/gloves/G = gloves
+			gloves_siemens_coeff = G.siemens_coefficient
+		if(species)
+			species_siemens_coeff = species.siemens_coeff
+		siemens_coeff = gloves_siemens_coeff * species_siemens_coeff
+	if(heart_attack)
+		if(shock_damage * siemens_coeff >= 1 && prob(25))
+			heart_attack = 0
+			if(stat == CONSCIOUS)
+				src << "<span class='notice'>You feel your heart beating again!</span>"
+	. = ..(shock_damage,source,siemens_coeff,safety,override,tesla_shock)

```

Comment:
This should be `. = ..()`, as the parent call operator provides the entire argument list by default.